### PR TITLE
Reduce logging output and refactor headers

### PR DESCRIPTION
**Pull request by Thaodan**
- Refactor handling of headers, only include whats needed
- Log categorized only when logging category is enabled. #33
- Add my self to authors, correct website

### CODE REVIEW

File: plasmoid/contents/config/main.xml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<!-- -*- xml -*-
+    SPDX-License-Identifier: GPL-3.0-or-later
+!-->
```
Comment (duvholt) [inline plasmoid/contents/config/main.xml:4]: This somehow broke the plasmoid config. Maybe a bad xml parser?

**Review by duvholt — CHANGES_REQUESTED** [review]
Thanks for cleaning up the logging and headers handling!

Could you also add a short note in the readme on how to enable debug logging?

File: plasmoid/contents/config/main.xml (hunk 1 above)
Comment (Thaodan) [reply plasmoid/contents/config/main.xml:4]: Could be that the `<?xml` line has to come first. Please try again.

**Review by duvholt — APPROVED** [review]
Thanks!

**Pull request merged**